### ISSUE TITLE
Logo for dunner

### ISSUE THREAD

**Issue by apoorvam** [leopardslab/dunner#135]
Dunner would love a logo for it!

**Comment by hv7214** [comment]
Hey, anyone up for the logo?. I was thinking to add this feature.

**Comment by agentmilindu** [comment]
please go ahead, you can send us your designs! We would love to see.

**Comment by hv7214** [comment]
What about this?
![logo](https://user-images.githubusercontent.com/42354803/69933883-4330bf80-14f6-11ea-9c32-fa74f408e826.png)

**Comment by apoorvam** [comment]
Good work!

Can you give us a description of the thoughts behind this logo? How does it relate to Containers/Task runner etc?

**Comment by hv7214** [comment]
I mainly focused on the runner keyword(so R), blue color kinda represents docker.

**Comment by hv7214** [comment]
@apoorvam  Is logo fine? or i should give another try?

**Comment by apoorvam** [comment]
@hv7214 Definitely it is some good work and good reasoning. But I personally not convinced that it can relate to Dunner very much in first glance because it highlights R. If possible you can give another try. However, we'll keep this logo also as an option until we finalize.

Would be great if logo has flavours of some/all of these: Containers/Docker/Golang/Dunner in it.

**Comment by hv7214** [comment]
Okay, i will another shot. Infact, now i'm also not much convinced with this logo.

**Comment by cjprogamer** [comment]
![logo](https://user-images.githubusercontent.com/41909955/71305192-cd957200-2384-11ea-8a29-0b7c3b0958d8.png)
Here is a logo !

**Comment by cjprogamer** [comment]
![logo](https://user-images.githubusercontent.com/41909955/71305214-12210d80-2385-11ea-8bc8-5a4636e1315f.png)
Here is a logo !

**Comment by cjprogamer** [comment]
Here is a logo ! 

![logo](https://user-images.githubusercontent.com/41909955/71305232-60cea780-2385-11ea-9b53-7f6e3e5c9e79.png)

**Comment by alpha2320** [comment]
> Here is a logo !
> 
> ![logo](https://user-images.githubusercontent.com/41909955/71305232-60cea780-2385-11ea-9b53-7f6e3e5c9e79.png)

nice design .... just something similar to it :
https://hackernoon.com/building-a-development-environment-for-golang-with-docker-3ccc620c4920
👍 💯

**Comment by cjprogamer** [comment]
> 
> nice design .... just something similar to it :
> https://hackernoon.com/building-a-development-environment-for-golang-with-docker-3ccc620c4920

Thnk you very much. Is there a problem with the logo ? (cuz it's also used by hackernoon ? )

**Comment by alpha2320** [comment]
> > nice design .... just something similar to it :
> > https://hackernoon.com/building-a-development-environment-for-golang-with-docker-3ccc620c4920
> 
> Thnk you very much. Is there a problem with the logo ? (cuz it's also used by hackernoon ? )

nooo.... just found something similar ....i am also a participant ... sometime hit me up!... ;)....
also do tell me how is mine looking?... will love to hear a review from u dude!
<img width="720" alt="Screenshot 2019-12-22 at 9 42 13" src="https://user-images.githubusercontent.com/58825698/71316978-f4949680-249f-11ea-9237-e8658977235a.png">

**Comment by chamiduu** [comment]
It's good, but I think It is more complex. Come up with a minimilistic logo ! ;-)

**Comment by chamiduu** [comment]
Here is an one ! 
![logo](https://user-images.githubusercontent.com/41909955/71343020-5f32ea00-2513-11ea-8056-91a5029159a3.png)

**Comment by hv7214** [comment]
Great work @cjprogamer!

**Comment by alpha2320** [comment]
![dunner_alpha](https://user-images.githubusercontent.com/58825698/71343373-a6f25980-2585-11ea-944b-71ff5f92365e.png)

**Comment by SufiyanPasta** [comment]
![Dunner-logo](https://user-images.githubusercontent.com/58876317/71401403-2b56e200-2643-11ea-9e68-123b3ae9190b.png)
![Dunner-logo (1)](https://user-images.githubusercontent.com/58876317/71401407-30b42c80-2643-11ea-868c-1e688d0741e6.png)

**Comment by SufiyanPasta** [comment]
how about this?

**Comment by Mohtashim1210** [comment]
![Dunner](https://user-images.githubusercontent.com/58835199/71475476-956db380-280a-11ea-9938-e8941a2b19f5.png)
How about this?

**Comment by apoorvam** [comment]
@Mohtashim1210 Can you give a brief justification of how your logo is suitable for Dunner? That can help us understand and give feedback.

Would recommend to include ideas of Docker, Task runner, Golang in the logo since they depict what dunner is related to.

**Comment by saadpasta** [comment]
![Dunner-logo (3)](https://user-images.githubusercontent.com/23307811/71540774-e4355d80-2970-11ea-82ee-271388f192ea.png)

**Comment by SufiyanPasta** [comment]
![Dunner-logo (3)](https://user-images.githubusercontent.com/58876317/71540798-1ba40a00-2971-11ea-8760-6b8a40443ea7.png)

**Comment by Dhanujanawanjana** [comment]
Dunner logo
![DUNNER LOGO2](https://user-images.githubusercontent.com/59271302/71607556-563fb980-2ba0-11ea-846a-57b53ca4a707.png)

**Comment by sharman13** [comment]
This is my logo
![Dunner logo](https://user-images.githubusercontent.com/58657244/71656667-2809d780-2d62-11ea-995b-254e156899fa.png)

**Comment by ghost** [comment]
![Dunner Logo](https://user-images.githubusercontent.com/59133890/71678514-8a3cf980-2daf-11ea-90b7-3e0900c242b4.png)

**Comment by mikutav** [comment]
![20200116_221405](https://user-images.githubusercontent.com/59904433/72544813-06563780-38ae-11ea-82ed-c41774ec5213.png)

My submission sir 
Royal shiv

**Comment by AF000001** [comment]
![Dunner](https://user-images.githubusercontent.com/59760341/72730460-6d7a3180-3bb7-11ea-86d8-bf028a80ef9a.png)

**Comment by AF000001** [comment]
![Dunner](https://user-images.githubusercontent.com/59760341/72730586-a87c6500-3bb7-11ea-9398-894302609c91.png)

**Comment by parasgathani7143** [comment]
![paras dunner](https://user-images.githubusercontent.com/42924011/72798072-47609a00-3c68-11ea-8894-a7946cb55c3a.png)

**Comment by parasgathani7143** [comment]
![ergfv](https://user-images.githubusercontent.com/42924011/72862273-95ba7b00-3cf2-11ea-821f-160d082221ce.png)

**Comment by parasgathani7143** [comment]
![alpha52](https://user-images.githubusercontent.com/42924011/72883342-252f5080-3d2a-11ea-9120-f857fbc79a53.png)

**Comment by AnirudhFarwaha** [comment]
![Logo](https://user-images.githubusercontent.com/59760341/72923911-f17a1800-3d75-11ea-9425-93ec1a8be1a2.png)

**Comment by kmehant** [comment]
Hello,
@apoorvam 
I just got to know Dunner and [Hasura](https://hasura.io/) got the same logo.
We might need to update it. As Google code-in is done, with many logo submissions for Dunner we might choose one and make it official ;)
Or Someone can have their designs here, I guess :)

**Comment by alpha2320** [comment]
> Hello,
> @apoorvam
> I just got to know Dunner and [Hasura](https://hasura.io/) got the same logo.
> We might need to update it. As Google code-in is done, with many logo submissions for Dunner we might choose one and make it official ;)
> Or Someone can have their designs here, I guess :)

![New Project (3)](https://user-images.githubusercontent.com/58825698/80565055-28af3380-8a0d-11ea-8bf4-ed8ef47108ea.png)
 how about this?

**Comment by apoorvam** [comment]
Right, we should update it. They have same currently because Dunner uses their gatsby template for website. 
We can shortlist some of the logos here, choose one and make some changes if required, then finalize Dunner logo!
